src/main/java/SYSC4806/Model/LearningOutcome.java

```diff
@@ -18,7 +15,11 @@
     @GeneratedValue(strategy= GenerationType.AUTO)
     private long id;
     private String name;
+
+    @ManyToMany
```

Comment:
In category, you have it listed as one-to-many. A learning outcome only maps to one category; this should be a many-to-one.